src/coprocessor/dag/builder.rs

```diff
@@ -63,6 +63,19 @@ impl DAGBuilder {
                         ))
                     })?;
                 }
+                ExecType::TypeAggregation => {
```

Comment:
Can we merge these two cases?

Reply:
Maybe better to not do this, because Fast and Slow have different checking logics, so it would be nice to make them separated and focused.

Reply:
Hmmm... what I mean is `ExecType::TypeAggregation | ExecType::TypeStreamAgg` and `ExecType::TypeAggregation`.

Reply:
Oh. Got it. Then we will need to repeat SimpleAggregation in two places (for both HashAgg and StreamAgg request).